# Define the goal
geometry_msgs/Pose[] frames
float32[] time
bool incremental
---
# Define the result
geometry_msgs/Pose final_error
float32 position_error_norm
float32 orientation_error_angle
---
# Define a feedback message
